config.js

```diff
@@ -5,7 +5,12 @@ const config = {
     titlePrefix: '',
     titleSuffix: ' | Prisma Docs',
   },
-  redirects: [],
+  redirects: [
+    {
+      from: '/',
+      to: '/',
```

Comment:
That looks like something we should not keep around.

Reply:
Leaving the comment here as well for context:

Gatsby usually asks for dummy data to be written in order to go through all the frontmatter values and other query values that may have been assigned and used for filters or any type of logic, like here: https://github.com/prisma/docs/actions/runs/3159217753/jobs/5142171081